ydb/core/http_proxy/http_req.cpp

```diff
@@ -462,6 +462,7 @@ namespace NKikimr::NHttpProxy {
 
         public:
             void Bootstrap(const TActorContext& ctx) {
+                PoolId = ctx.SelfID.PoolID();
```

Comment:
Not used variable

Reply:
It used here https://github.com/ydb-platform/ydb/pull/11244/files#diff-f22aa19d1f64f0a3be66206c685ea066c13a301980a1d3c3dde7c568821ad6abR551